graphix/visualization.py

```diff
@@ -1015,8 +1014,7 @@ def get_pos_from_flow(self, f: dict[int, int], l_k: dict[int, int]) -> dict[int,
         for i, k in enumerate(start_nodes):
             pos[k][1] = i
             while k in f.keys():
-                k = list(f[k])[0]
-                pos[k][1] = i
+                pos[next(iter(f[k]))][1] = i
```

Comment:
This change introduces an end-less while-loop by not updating `k`. I propose a fix and a test to prevent regression here: https://github.com/thierry-martinez/graphix/commit/afe40cdfb84df1a64effe95e7505f8d2612af7d9

Reply:
I see. Let me cherry-pick it!